jwala-webapp/src/main/webapp/resources/js/react/WebAppConfig.js

```diff
@@ -20,7 +20,7 @@ var WebAppConfig = React.createClass({
                    <RDataTable ref="dataTable"
                                tableIndex="id.id"
                                colDefinitions={[{key: "id", isVisible: false},
-                                                {key: "secure", renderCallback: this.renderSecureColumn, sortable: false},
+                                                {key: "secure", renderCallback: this.renderSecureColumn, sortable: false, isVisible: false},
```

Comment:
Can you revert this ? Because we don't need this change anymore. Remember one of the reasons for the change request is to limit the change in RDataTable.js ?